### PR TITLE
fix: ensure assignee is checked in userUnassigned handler

### LINKED ISSUE #74: User un-assign should close its linked pull-requests
Disqualify needs to close the pull

_Originally posted by @0x4007 in https://github.com/ubiquity-os-marketplace/daemon-merging/issues/22#issuecomment-2452913872_

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves #74
QA: https://github.com/Meniole/command-start-stop/pull/45

### CODE REVIEW

**Comment by gentlementlegen** [comment]
@Keyrxng if you want to take a look.

File: src/handlers/user-start-stop.ts [hunk 1]
```diff
@@ -33,6 +33,12 @@ export async function userSelfAssign(context: Context<"issues.assigned">): Promi
   const { issue } = payload;
   const deadline = getDeadline(issue.labels);
 
+  // We avoid posting a message if the bot is the actor to avoid double posting
+  if (payload.sender.type === "Bot") {
```
Comment (Keyrxng) [inline src/handlers/user-start-stop.ts:37]: You could merge with the `if` below since they are the same response

**Review by Keyrxng — APPROVED** [review]
Seems to work as intended. 

Previous problem was the sender was used to check for PRs now assignee is used which makes sense

File: src/handlers/user-start-stop.ts (hunk 1 above)
Comment (gentlementlegen) [reply src/handlers/user-start-stop.ts:37]: Will do so and merge.

**Pull request merged**